app/schemas/pl.droidevs.books.app.BookDataBase/2.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Is this file autogenerated? If yes, does it have to be in the repository?

Reply:
Yes, it is autogenerated, when changes to the database are made. It is generated when you add a simple command in the gradle script. I was helpful to me in order to see if changes were applied. However Its true, It doesnt need to be included.